packages/lodestar/src/network/peers/peerManager.ts

```diff
@@ -113,6 +113,8 @@ export class PeerManager {
 
   private seenPeers = new Set<string>();
 
+  private firstHearbeat = true;
```

Comment:
We could generalize this into N seconds after starting discv5, and fine tune accordingly